origin-notifications/src/config/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Add a TODO. For productionizing this we'll need to use env variables (DATA_USERNAME, DATABASE_PASSWORD, etc...)